packages/sdk/src/waku/waku.ts

```diff
@@ -106,7 +107,16 @@ export class WakuNode implements IWaku {
     this.health = getHealthManager();
 
     if (protocolsEnabled.store) {
-      const store = wakuStore(this.connectionManager);
+      let peerIdStr: PeerIdStr | undefined;
+      if (options.store?.peer) {
+        this.connectionManager
+          .dialPeer(options.store.peer, [StoreCodec])
+          .catch((e) => {
+            log.error("Failed to dial store peer", e);
+          });
+      }
+
+      const store = wakuStore(this.connectionManager, peerIdStr);
```

Comment:
this is wrong, `peerIdStr` does nothing here

Reply:
It is used to initialise Store with the protocol to use since we need to get the peer when we send a store request. Changed the `peerIdStrToUse` to `options: Partial<StoreProtocolOptions>`